src/index.html

```diff
@@ -19,13 +19,15 @@
   <body>
     <input
       type="text"
-      data-qa="keypress"
+      class="search-field search--lange"
```

Comment:
There is a typo in the class name `search--lange`. It should probably be `search--large` to match a common naming convention for sizes. Please verify the correct class name in your CSS file.